scripts/gen_regex.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import unicodedata
+import chardata
+import pathlib
+
+DATA_PATH = pathlib.Path(resource_filename('wordfreq', 'data'))
+
+def _emoji_char_class():
```

Comment:
General comment: isn't two lines between functions (outside of classes) standard in pep8?